shoop/admin/modules/orders/static_src/create/reducers/customer.js

```diff
@@ -31,7 +31,13 @@ function setAddressProperty(state, {payload}) {
 }
 
 export default handleActions({
-    setCustomer: ((state, {payload}) => payload),
+    setCustomer: ((state, {payload}) => _.assign(state, {
```

Comment:
This could just do `_.assign(state, payload)` to copy all properties from `payload` to the `state` without having to enumerate all of them by hand, couldn't it?